VERSIONS.md

```diff
@@ -2,16 +2,16 @@
 
 See below for a full list of minimum system requirements for historical versions of the Core Components:
 
-Core Components | Extension | AEM 6.4 | AEM 6.3 | Java
-----------------|-----------|---------|---------|------
-[2.3.2](https://github.com/adobe/aem-core-wcm-components/releases/tag/core.wcm.components.reactor-2.3.2) | 1.0.12 | 6.4.2.0 | 6.3.3.0 | 1.8
-[2.3.0](https://github.com/adobe/aem-core-wcm-components/releases/tag/core.wcm.components.reactor-2.3.0) | 1.0.10 | 6.4.2.0 | 6.3.3.0 | 1.8
-[2.2.2](https://github.com/adobe/aem-core-wcm-components/releases/tag/core.wcm.components.reactor-2.2.2) | 1.0.10 | 6.4.2.0 | 6.3.3.0 | 1.8
-[2.2.0](https://github.com/adobe/aem-core-wcm-components/releases/tag/core.wcm.components.reactor-2.2.0) | 1.0.8 | 6.4.2.0 | 6.3.3.0 | 1.8
-[2.1.0](https://github.com/adobe/aem-core-wcm-components/releases/tag/core.wcm.components.reactor-2.1.0) | 1.0.6 | 6.4.2.0 | 6.3.3.0 | 1.8
-[2.0.6](https://github.com/adobe/aem-core-wcm-components/releases/tag/core.wcm.components.reactor-2.0.6), [2.0.8](https://github.com/adobe/aem-core-wcm-components/releases/tag/core.wcm.components.reactor-2.0.8) | 1.0.2, 1.0.4 | 6.4.0.0 | 6.3.2.0 | 1.8
-[2.0.4](https://github.com/adobe/aem-core-wcm-components/releases/tag/core.wcm.components.reactor-2.0.4) | 1.0.0 | 6.4.0.0 | 6.3.2.0 | 1.8
-[2.0.0](https://github.com/adobe/aem-core-wcm-components/releases/tag/core.wcm.components.reactor-2.0.0) | sandbox/preview | 6.4.0.0 | 6.3.2.0 | 1.8
-[1.1.0](https://github.com/adobe/aem-core-wcm-components/releases/tag/core.wcm.components.reactor-1.1.0) | sandbox/preview | 6.4.0.0 | 6.3.1.0 | 1.8
-[1.0.4](https://github.com/adobe/aem-core-wcm-components/releases/tag/core.wcm.components.reactor-1.0.4), [1.0.6](https://github.com/adobe/aem-core-wcm-components/releases/tag/core.wcm.components.reactor-1.0.6) | - | 6.4.0.0 | 6.3.0.0 | 1.8
-[1.0.0](https://github.com/adobe/aem-core-wcm-components/releases/tag/core.wcm.components.reactor-1.0.0), [1.0.2](https://github.com/adobe/aem-core-wcm-components/releases/tag/core.wcm.components.all-1.0.2) | - | 6.4.0.0 | 6.3.0.0 | 1.7
+Core Components | Extension | AEM 6.5 | AEM 6.4 | AEM 6.3 | Java
+----------------|-----------|---------|---------|---------|------
+[2.3.2](https://github.com/adobe/aem-core-wcm-components/releases/tag/core.wcm.components.reactor-2.3.2) | 1.0.12 | 6.5.0.0 | 6.4.2.0 | 6.3.3.0 | 1.8
+[2.3.0](https://github.com/adobe/aem-core-wcm-components/releases/tag/core.wcm.components.reactor-2.3.0) | 1.0.10 | - | 6.4.2.0 | 6.3.3.0 | 1.8
```

Comment:
@bohnertchris - 2.1.0 -> 2.3.2 should be supported/usable on 6.5.0.0, right?

Reply:
We don't specifically test this, right?

Reply:
The way it's written now suggests that an upgrade to 2.3.2+ is required when using AEM 6.5, which just isn't true.